docs/05_community-developer-tools/index.md

```diff
@@ -76,7 +78,9 @@ If you would like to submit a tool to be added to the list, please fill in [this
 
 * [Scatter](https://github.com/GetScatter/ScatterDesktop): Scatter desktop is a cutting-edge wallet as well as a global local-machine authentication provider.
 
-* [Portis](https://docs.portis.io/#/): Portis is a "Non-Custodial Blockchain Wallet that Makes Apps Simple for Everyone"
+* [Portis](https://docs.portis.io/#/): Portis is a "Non-Custodial Blockchain Wallet that Makes Apps Simple for Everyone".
```

Comment:
How about we get rid of the marketing description and use something informative like:

_A non-custodial blockchain wallet that provides key management and smart contract interaction using end-to-end encryption_.

I got the second part of the text from the [Portis Whitepaper](https://assets.portis.io/white-paper/latest.pdf)